sdk/tests/conformance/extensions/ext-float-blend.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>

```

Comment:
You didn't add this file to the test list file.
Also, I would really prefer merging the webgl1 and 2 versions of this test, if it's not toooo much work.

Reply:
:man_facepalming: My bad. A new pull request is here: https://github.com/KhronosGroup/WebGL/pull/2900